docs/source/support/faq.rst

```diff
@@ -2,6 +2,12 @@
 Frequently Asked Questions
 ==========================
 
+Is there a Wiki?
+----------------
+
+Yes - https://github.com/nucypher/wiki/wiki
```

Comment:
We should discuss if this is something we want to actively maintain. Currently, we don't but maybe we should, especially if we're going to surface it in the FAQ.

Maybe some part of the protocol repo should live here? @arjunhassard

Reply:
I'm In favor of removing this link.

Reply:
✔️ - removed